src/Callout/CalloutMoreFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import React, { ElementType, FC, useState } from 'react';
+import { Button } from '../Button/Button';
+import { Icon } from '../Icon/Icon';
+
+export interface CalloutMoreFooterProps {
+  /** Utilizzarlo in caso si utilizzo di componenti personalizzati */
+  tag?: ElementType;
+  /** Classi aggiuntive da usare per il componente */
+  className?: string;
+  fileUrl?: string;
+}
+
+export const CalloutMoreFooter: FC<CalloutMoreFooterProps> = ({
+  tag = 'div',
+  className,
+  fileUrl = '#',
+  children,
+  ...attributes
+}) => {
+  const Tag = tag;
+  const classes = classNames('collapse-div', className);
+
+  const [isOpen, setOpen] = useState(false);
+
+  return (
+    <Tag {...attributes} className={classes}>
+      <div className='collapse-header' id='heading1'>
+        <Button
+          color='primary'
+          className='callout-more-toggle'
+          onClick={() => setOpen(!isOpen)}
+          aria-controls='collapse1'
+          aria-expanded={isOpen}
+          style={{ fontFamily: 'var(--bs-font-sans-serif)', fontSize: '1rem' }}
```

Comment:
if possible I would avoid styles, with exceptions

Reply:
I don't like style myself, but unfortunately the button seems to inherit the font settings from `callout-div`. The CSS variable from the `btn` class seems to have an empty value:
<img width="224" alt="image" src="https://github.com/italia/design-react-kit/assets/35468654/0d0ec780-0fc1-4088-a8a7-cbdb3a4b1d89">

Also, I overrode the `fontSize` because `btn` one is set to a fixed `16px`